coalib/bears/Bear.py

```diff
@@ -318,8 +354,13 @@ def run_bear_from_section(self, args, kwargs):
             self.warn('The bear {} cannot be executed.'.format(
                 self.name), str(err))
             return
-        if self.debugger:
+        if self.profile and self.debugger:
+            raise ValueError('Invalid argument passed, run debugger or '
+                             'profiler one at a time.')
```

Comment:
the exception should be inside `__init__`. Don't allow to create bears that can't run.

Reply:
Makman we are not handeling the `ValueError` in [processing.py](https://github.com/coala/coala/blob/master/coalib/processes/Processing.py#L336) should I handle the `ValueError` so that other bear can initialize and  run?

Reply:
>Don't allow to create bears that can't run.

Nvm IMO halting the further execution will be good